### ISSUE TITLE
'doit test` add tests dependencies installation?

### ISSUE THREAD

**Issue by yakutovicha** [aiidalab/aiidalab-docker-stack#284]
The [readme](https://github.com/aiidalab/aiidalab-docker-stack/tree/docker-buildx-bake#build-images-locally) suggest to do the following:
```
1. `conda env create -f environment.yml`
2. `conda activate aiidalab-docker-stack`
3. `doit build`
4. `doit tests`
```
But if one follows those steps blindly, `doit tests` will fail because of missing dependencies. I suggest we install dependencies as part of `doit tests`, or we add a small text that suggests installing dependencies before running `doit tests`.

**Comment by yakutovicha** [comment]
very efficient :)

**Comment by csadorf** [comment]
Fixed in 38c9a6f2a8d99a266c7da5be0005e87f9a0ebdeb .
